src/frontend/calendar/cell.js

```diff
@@ -29,7 +29,11 @@ function CalendarCell( {
 	const restOfEvents = dayEvents.slice( MAX_EVENTS );
 
 	return (
-		<td className="wporg-meeting-calendar__cell">
+		<td
+			className={ `wporg-meeting-calendar__cell ${
+				isToday( date ) ? 'today' : ''
```

Comment:
Status classes should use `is-` prefix

```suggestion
				isToday( date ) ? 'is-today' : ''
```

Reply:
Totally. Thanks!